src/ArticleHolder.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import Articalizer from "./Articalizer";
+
+export default function ArticleHolder() {
```

Comment:
I'd probably call this something else. It's holding all the Articles rather than a single Article, which is what the description currently indicates at first glance.